clang/include/clang/Basic/BuiltinsAssigner.def

```diff
@@ -59,5 +59,16 @@ BUILTIN(__builtin_assigner_exit_check, "vb", "n")
 
 BUILTIN(__builtin_assigner_print_native_pallas_field, "vg1", "n")
 
+BUILTIN(__builtin_assigner_fri_lin_inter, "g1g1g1g1g1", "n")
```

Comment:
Let's wrap them in comment that these builtins are related to Placeholder, both here and in llvm